src/txn/txn_rollback_to_stable.c

```diff
@@ -8,6 +8,9 @@
 
 #include "wt_internal.h"
 
+/* Enable rollback to stable verbose messaging during recovery. */
+#define WT_VERB_RECOVERY_RTS_FLAG(conn) \
+    (F_ISSET((conn), WT_CONN_RECOVERING) ? WT_VERB_RECOVERY | WT_VERB_RTS : WT_VERB_RTS)
```

Comment:
I have two minor suggestions for this macro:
1. I think it should take a `session` and then just do the `S2C` inside this macro rather than every caller needing to do that.
2. I don't think `_FLAG` adds much to the name. `WT_VERB_RECOVERY_RTS` conveys the same to me and is a bit shorter.

Reply:
> I think it should take a session and then just do the S2C inside this macro rather than every caller needing to do that.

That is how Sid did it the first time and I suggested we pass connection if that was what the macro required. @Siddhartha8899 , I don't mind if you revert back to passing session to the macro.

Reply:
I did not see those comments on the outdated diff this morning. Either way, if you have a strong preference. The diff made it look less clean, but I can definitely see where your POV is coming from.